<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InGame</class>
 <widget class="QMainWindow" name="InGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InGame</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="chessBoard">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bPawn1">
    <property name="geometry">
     <rect>
      <x>27</x>
      <y>75</y>
      <width>38</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="boardEdge">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>420</width>
      <height>420</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bPawn2">
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>75</y>
      <width>38</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bPawn3">
    <property name="geometry">
     <rect>
      <x>127</x>
      <y>75</y>
      <width>38</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bPawn4">
    <property name="geometry">
     <rect>
      <x>177</x>
      <y>75</y>
      <width>38</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bPawn5">
    <property name="geometry">
     <rect>
      <x>227</x>
      <y>75</y>
      <width>38</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bPawn6">
    <property name="geometry">
     <rect>
      <x>277</x>
      <y>75</y>
      <width>38</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bPawn7">
    <property name="geometry">
     <rect>
      <x>327</x>
      <y>75</y>
      <width>38</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bPawn8">
    <property name="geometry">
     <rect>
      <x>377</x>
      <y>75</y>
      <width>38</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bRook1">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>25</y>
      <width>41</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bRook2">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>25</y>
      <width>41</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bKnight1">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>25</y>
      <width>41</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bKnight2">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>25</y>
      <width>41</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bBishop1">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>25</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bBishop2">
    <property name="geometry">
     <rect>
      <x>274</x>
      <y>25</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bKing">
    <property name="geometry">
     <rect>
      <x>224</x>
      <y>25</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bQueen">
    <property name="geometry">
     <rect>
      <x>172</x>
      <y>25</y>
      <width>49</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="wPawn2">
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>325</y>
      <width>38</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="wPawn3">
    <property name="geometry">
     <rect>
      <x>127</x>
      <y>325</y>
      <width>38</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="wPawn1">
    <property name="geometry">
     <rect>
      <x>27</x>
      <y>325</y>
      <width>38</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="wPawn5">
    <property name="geometry">
     <rect>
      <x>227</x>
      <y>325</y>
      <width>38</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="wPawn6">
    <property name="geometry">
     <rect>
      <x>277</x>
      <y>325</y>
      <width>38</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="wPawn7">
    <property name="geometry">
     <rect>
      <x>327</x>
      <y>325</y>
      <width>38</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="wPawn4">
    <property name="geometry">
     <rect>
      <x>177</x>
      <y>325</y>
      <width>38</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="wPawn8">
    <property name="geometry">
     <rect>
      <x>377</x>
      <y>325</y>
      <width>38</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="wKing">
    <property name="geometry">
     <rect>
      <x>224</x>
      <y>375</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="wBishop2">
    <property name="geometry">
     <rect>
      <x>274</x>
      <y>375</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="wBishop1">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>375</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="wKnight2">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>375</y>
      <width>41</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="wKnight1">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>375</y>
      <width>41</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="wRook1">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>375</y>
      <width>41</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="wQueen">
    <property name="geometry">
     <rect>
      <x>172</x>
      <y>375</y>
      <width>49</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="wRook2">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>375</y>
      <width>41</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>boardEdge</zorder>
   <zorder>chessBoard</zorder>
   <zorder>bPawn1</zorder>
   <zorder>bPawn2</zorder>
   <zorder>bPawn3</zorder>
   <zorder>bPawn4</zorder>
   <zorder>bPawn5</zorder>
   <zorder>bPawn6</zorder>
   <zorder>bPawn7</zorder>
   <zorder>bPawn8</zorder>
   <zorder>bRook1</zorder>
   <zorder>bRook2</zorder>
   <zorder>bKnight1</zorder>
   <zorder>bKnight2</zorder>
   <zorder>bBishop1</zorder>
   <zorder>bBishop2</zorder>
   <zorder>bKing</zorder>
   <zorder>bQueen</zorder>
   <zorder>wRook1</zorder>
   <zorder>wRook2</zorder>
   <zorder>wBishop1</zorder>
   <zorder>wPawn1</zorder>
   <zorder>wPawn7</zorder>
   <zorder>wPawn8</zorder>
   <zorder>wBishop2</zorder>
   <zorder>wKnight2</zorder>
   <zorder>wPawn2</zorder>
   <zorder>wPawn5</zorder>
   <zorder>wPawn6</zorder>
   <zorder>wPawn4</zorder>
   <zorder>wKnight1</zorder>
   <zorder>wPawn3</zorder>
   <zorder>wQueen</zorder>
   <zorder>wKing</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
